achyut/src/main/java/com/lftechnology/runner/Menu.java

```diff
@@ -0,0 +1,22 @@
+package com.lftechnology.runner;
+
+public class Menu {
+
+    public Menu() {
+
+    }
+
+    public void showmenu() {
+        System.out.println("Please hit the options below to play with us");
+        System.out.println("Options   Program");
+        System.out.println("1         Salutation");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)